packages/webpack.config.js

```diff
@@ -82,6 +84,18 @@ function getConfig(pkg) {
           },
         ],
       }),
+      new JsonPostProcessPlugin({
+        matchers: [
+          {
+            // Add H5Web's dependencies to the package's output `package.json`
+            matcher: /^package\.json$/u,
+            action: (json) => ({
+              ...json,
+              dependencies: omit(rootPkg.dependencies, ['react', 'react-dom']),
```

Comment:
I add all the dependencies to the `package.json` of `@h5web/lib` and `@h5web/app`. This is so we don't have to keep the version numbers in sync in three different `package.json` files...

Reply:
I see.

Could we use `.filter` to avoid the `lodash` dependency ?

Reply:
Hmm it's quite convenient, and the dependency is only in `packages/package.json`, so it doesn't really matter.